singer_sdk/typing.py

```diff
@@ -402,6 +411,13 @@ def to_dict(self) -> dict:
             type_dict.update({"default": self.default})
         if self.description:
             type_dict.update({"description": self.description})
+        if self.secret:
+            type_dict.update(
+                {
+                    "secret": True,
```

Comment:
`"sensitive"` is used elsewhere. If that was the intention, maybe it's a good idea to use a `JSON_SCHEMA_SENSITIVE_KEY = "sensitive"` constant that's imported where it's needed?

Reply:
Good catch. I switched during development from 'sensitive' to 'secret' on the basis that _lots_ of data is "sensitive" that is not also a secret in the technical sense of secret-management. (Revenue, PII, and lots of other fields, for instance can be sensitive but not 'writeOnly' or 'secret' in the annotation type.)

Reply:
Done.